webservices/common/models/itemized.py

```diff
@@ -269,7 +297,7 @@ def pdf_url(self):
         return None
 
 
-class ScheduleE(BaseItemized):
+class ScheduleE(BaseItemized, PdfMixin):
```

Comment:
Remember about Python's MRO (Method Resolution Order) - you'll want the base class you intended to use as the last thing listed here and any mixins or other abstract classes before it:

```python
class ScheduleE(PdfMixin, BaseItemized)
```